<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="10307" started-at="2018-11-19T16:42:02Z" finished-at="2018-11-19T16:42:12Z">
    <groups>
    </groups>
    <test name="Test API" duration-ms="10307" started-at="2018-11-19T16:42:02Z" finished-at="2018-11-19T16:42:12Z">
      <class name="MavenFramwwork.Demo">
        <test-method status="PASS" signature="createPlaceAPI()[pri:0, instance:MavenFramwwork.Demo@402bba4f]" name="createPlaceAPI" duration-ms="1838" started-at="2018-11-19T16:42:08Z" finished-at="2018-11-19T16:42:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPlaceAPI -->
      </class> <!-- MavenFramwwork.Demo -->
      <class name="MavenFramwwork.PostRequest">
        <test-method status="PASS" signature="createPlaceAPI()[pri:0, instance:MavenFramwwork.PostRequest@795cd85e]" name="createPlaceAPI" duration-ms="448" started-at="2018-11-19T16:42:12Z" finished-at="2018-11-19T16:42:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPlaceAPI -->
      </class> <!-- MavenFramwwork.PostRequest -->
      <class name="MavenFramwwork.AddnDeletetheRequest">
        <test-method status="PASS" signature="getData()[pri:0, instance:MavenFramwwork.AddnDeletetheRequest@498d318c]" name="getData" is-config="true" duration-ms="31" started-at="2018-11-19T16:42:02Z" finished-at="2018-11-19T16:42:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getData -->
        <test-method status="FAIL" signature="addNdeleteID()[pri:0, instance:MavenFramwwork.AddnDeletetheRequest@498d318c]" name="addNdeleteID" duration-ms="1723" started-at="2018-11-19T16:42:10Z" finished-at="2018-11-19T16:42:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
Expected status code <201> but was <500>.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <201> but was <500>.

at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:60)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:235)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:483)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:655)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:123)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:131)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:119)
at MavenFramwwork.AddnDeletetheRequest.addNdeleteID(AddnDeletetheRequest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNdeleteID -->
      </class> <!-- MavenFramwwork.AddnDeletetheRequest -->
      <class name="MavenFramwwork.APIGetRequest">
        <test-method status="PASS" signature="getPlaceAPI()[pri:0, instance:MavenFramwwork.APIGetRequest@6e171cd7]" name="getPlaceAPI" duration-ms="6154" started-at="2018-11-19T16:42:02Z" finished-at="2018-11-19T16:42:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPlaceAPI -->
      </class> <!-- MavenFramwwork.APIGetRequest -->
    </test> <!-- Test API -->
  </suite> <!-- Suite -->
</testng-results>
